packages/apps/docs/src/docs/build/pact/concepts.md

```diff
@@ -422,13 +422,23 @@ This "scoping" allows the signer to safely call untrusted code. For instance, in
 
 With that sender's signature has `(GAS)` added to it, it is scoped within gas payments in the coin contract only. Third-party code is prohibited from accessing that account during the transaction.
 
-### Signatures and Managed Capabilities
+### Signatures and managed capabilities
 
 Signature capabilities are also a mechanism to _install_ capabilities, but only if that capability is _managed_. "Vanilla" capabilities are just tickets to show before you try some protected operation, but _managed_ capabilities are able to _change the state_ of a capability as it is brought into and out of scope. The ticket metaphor breaks down here, as this is now a dynamic object that mediates whether capabilities are acquired.
 
 If a signer attaches a managed capability to their signature list, the capability is "installed", which is not the same as "granted" or "acquired": if the capability's predicate function allows this signer to install the capability, the installed version will then govern any code needing the capability to unlock some protected operation, by means of a _manager function_.
 
-#### Capability management with a manager function
+### Verifiers and capabilities
+
+Since Pact 4.11 ([KIP 0028](https://github.com/kadena-io/KIPs/pull/57)), capabilities can be installed by *verifiers*, which are also
+scoped to those capabilities that they install. A verifier is a named plugin external
+to Pact that, given some proof value—similar to a signature—can grant capabilities to perform some action.
```

Comment:
```suggestion
to Pact that, given some proof value (for example, a signature), can grant capabilities to perform some action.
```

Reply:
I think maybe somewhere in the middle is best here, because signatures do totally work here, but also other things analogous to signatures like ZK proofs.